extensions/kind/package.json

```diff
@@ -41,6 +41,12 @@
           "default": 9443,
           "scope": "KubernetesProviderConnectionFactory",
           "description": "HTTPS Port"
+        },
+        "kind.cluster.creation.ingress": {
+          "type": "boolean",
+          "default": true,
+          "scope": "KubernetesProviderConnectionFactory",
+          "description": "Setup an ingress controller"
```

Comment:
maybe say that it'll be contour ? (like within parenthesis) 

Also is that mid term we will offer a drop-down with more than Contour ?

Reply:
Updated property description